### PR TITLE
Details : Interactions

### PR DESCRIPTION
**Pull request by Cocatrix**

### CODE REVIEW

File: src/pages/details-contact/details-contact.ts [hunk 1]
```diff
@@ -92,12 +112,25 @@ export class DetailsContactPage {
...
+      this.favoriteButtonLabel = 'REMOVE_FROM_FAVORITES';
+    } else {
+      this.favoriteButtonLabel = 'ADD_TO_FAVORITES';
+    }
+
     actionSheet.present().then();
   }
 
+  deleteContact() {
+    this.contactsProvider.delete(this.contact.wsId);
```
Comment (newatox) [inline src/pages/details-contact/details-contact.ts]: You need to handle errors the provider might send you

File: src/pages/details-contact/details-contact.ts [hunk 2]
```diff
@@ -92,12 +112,25 @@ export class DetailsContactPage {
...
+    if (this.contact.isFavorite) {
+      this.favoriteButtonLabel = 'REMOVE_FROM_FAVORITES';
+    } else {
+      this.favoriteButtonLabel = 'ADD_TO_FAVORITES';
+    }
+
     actionSheet.present().then();
   }
 
+  deleteContact() {
```
Comment (newatox) [inline src/pages/details-contact/details-contact.ts]: Add an ionic alertcontroller for delete confirmation

File: src/pages/details-contact/details-contact.ts [hunk 3]
```diff
@@ -106,6 +139,36 @@ export class DetailsContactPage {
...
+          },
+        };
+        this.sms.send(this.contact.phone, message, options);
+        break;
+      case 'Emailing':
+        const body = message;
+        const email = {
+          body,
+          to: this.contact.email,
+          subject: 'Nouvelles',
```
Comment (newatox) [inline src/pages/details-contact/details-contact.ts]: This string should be internationalized

File: src/pages/details-contact/details-contact.ts (hunk 2 above)
Comment (Cocatrix) [reply src/pages/details-contact/details-contact.ts]: My bad. Done in next commit

File: src/pages/details-contact/details-contact.ts (hunk 1 above)
Comment (Cocatrix) [reply src/pages/details-contact/details-contact.ts]: Done

File: src/pages/details-contact/details-contact.ts (hunk 3 above)
Comment (Cocatrix) [reply src/pages/details-contact/details-contact.ts]: Yes ! Done

**Pull request merged**